pymc3/tests/test_distributions_random.py

```diff
@@ -469,6 +463,34 @@ def seeded_discrete_weibul_rng_fn(self):
     ]
 
 
+class TestPareto(BaseTestDistribution):
+    pymc_dist = pm.Pareto
+    pymc_dist_params = {"alpha": 3.0, "m": 2.0}
+    expected_rv_op_params = {"alpha": 3.0, "m": 2.0}
+    reference_dist_params = {"alpha": 3.0, "scale": 2.0}
```

Comment:
`reference_dist_params` should be "b" and "scale" I think

Reply:
Instead of `seeded_numpy_distribution_builder`, I should use `seeded_scipy_distribution_builder` to match the params, right?

Reply:
Yes, you're right. Aesara is using scipy for the Pareto